backend/lib/edgehog/containers/manual_actions/run_ready_actions.ex

```diff
@@ -0,0 +1,43 @@
+#
+# This file is part of Edgehog.
+#
+# Copyright 2024 SECO Mind Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+defmodule Edgehog.Containers.ManualActions.RunReadyActions do
+  @moduledoc false
+
+  use Ash.Resource.ManualUpdate
+
+  alias Edgehog.Containers
+
+  require Ash.Query
+
+  @impl Ash.Resource.ManualUpdate
+  def update(changeset, _opts, _context) do
+    deployment = changeset.data
+
+    with {:ok, deployment} <- Ash.load(deployment, :ready_actions) do
+      Enum.reduce_while(deployment.ready_actions, {:ok, deployment}, fn action, _acc ->
+        case Containers.run_ready_action(action) do
+          {:ok, _ready_action} -> {:cont, {:ok, deployment}}
+          {:error, reason} -> {:halt, {:error, reason}}
+        end
+      end)
```

Comment:
Maybe here we don't actually want to halt if one ready action fails, I think it would be better to just run the ready action and let them crash if something happens
```suggestion
      _ready_actions = Enum.map(deployment.ready_actions, &Containers.run_ready_action/1))
      {:ok, deployment}
```